.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
```

Comment:
Travis's main docs suggest you want that when using the `docker` service:

https://docs.travis-ci.com/user/docker/